test/scripts/base/can_find_dlt_viewer.py

```diff
@@ -14,6 +14,6 @@ def find_service(service_name, install_dir):
 
 
 if __name__ == "__main__":
-    service_name = "dlt_viewer"
-    if not find_service(service_name, "/usr/bin"):
-        raise Exception("Could not find dlt_viewer installed")
+    SERVICE_NAME = "dlt_viewer"
+    if not find_service(SERVICE_NAME, "/usr/bin"):
+        raise Exception("Could not find " + SERVICE_NAME + " installed")
```

Comment:
Also not PEP8, but it seems to be a reasonable change

Reply:
Since this is kind of a main function, or at least SERVICE_NAME is not a constant for the entire program, we should keep lower case here.

Alternatively, move the SERVICE_NAME variable outside the the if statement (making it a constant)

Reply:
Well, to clarify my comment, the last line (the one I commented on) is not about PEP8.
However, the use of capitals for

Reply:
Oh, I guess you mean the function parameter with the same name. 
```find_service(service_name, ...```

Since it is a different local variable, all the more reason to not use the same name in the calling function?  

I agree all of this commit is about nitpicks, so we can nuke it if you want.  It's just what a PEP8 check reported, (plus a few other things that were slipped in).

Reply:
Since `SERVICE_NAME` is not meant to be a constant so maybe we should move the contents of the if statement to a proper function called `main`.

```python
def main():
    service_name = "dlt_viewer"
    if not find_service(service_name, "/usr/bin"):
        raise Exception("Could not find " + service_name + " installed")

if __name__ == "__main__":
    main()
``` 

I guess it is starting to get a bit silly since the file is so tiny :smile: